tests/test_openjdk.py

```diff
@@ -210,14 +212,21 @@ def test_jdk_cassandra(container_per_test):
         f"tar xzvf apache-cassandra-{cassandra_version}-bin.tar.gz >/dev/null",
     )
 
+    cassandra_directory = f"cd apache-cassandra-{cassandra_version}/"
```

Comment:
Nitpick: this is not a directory but a `chdir` command -> the variable name could be improved
```suggestion
    cd_to_cassandra_dir_cmd = f"cd apache-cassandra-{cassandra_version}/"
```

Reply:
thanks, fixed